### PR TITLE
create directory for uploads

### PR DESCRIPTION
**Pull request by anegowska**
method which create directory "club-manager-uploads" for upload Hall images

### CODE REVIEW

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java [hunk 1]
```diff
@@ -28,18 +29,22 @@ public String getUploadImageFilesPath() throws IOException {
...
+        Path path = Paths.get(getUploadImageFilesPath());
+
         String fileName = Paths.get(filePart.getSubmittedFileName()).getFileName().toString();
 
         if(fileName == null || fileName.isEmpty()){
             throw new HallImageNotFound(NO_USER_IMAGE_HAS_BEEN_UPLOADED);
         }
 
-        File file = new File(getUploadImageFilesPath() + fileName);
+        File file = new File(getUploadImageFilesPath() + "/" + fileName);
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java]: MINOR Remove this hard-coded path-delimiter. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1075)

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java [hunk 2]
```diff
@@ -48,6 +53,17 @@ public File uploadImageFile(Part filePart) throws IOException, HallImageNotFound
 
     }
 
+    private static void createDirectory(Path path) {
+        if (!Files.exists(path)) {
+            try {
+                Files.createDirectory(path);
+            } catch (IOException e) {
+                e.printStackTrace();
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java]: MINOR Use a logger to log this exception. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1148)

**Comment by isacademyci** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java [hunk 3]
```diff
@@ -48,6 +57,18 @@ public File uploadImageFile(Part filePart) throws IOException, HallImageNotFound
 
     }
 
+    private static void createDirectory(Path path) {
+        if (!Files.exists(path)) {
+            try {
+                Files.createDirectory(path);
+            } catch (IOException e) {
+                LOG.warn("Directory not created");
+                e.printStackTrace();
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java]: wyrzuc pls prntStackTrace

File: ClubManagerWeb/src/main/resources/settings.properties [hunk 4]
```diff
@@ -1 +1,2 @@
-Upload.Path.Images = /tmp/club-manager-uploads/
\ No newline at end of file
+Upload.Path.Images = /home/tmp/club-manager-uploads
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/resources/settings.properties]: niech zostanie /tmp

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java [hunk 5]
```diff
@@ -28,18 +33,22 @@ public String getUploadImageFilesPath() throws IOException {
...
+        Path path = Paths.get(getUploadImageFilesPath());
+
         String fileName = Paths.get(filePart.getSubmittedFileName()).getFileName().toString();
 
         if(fileName == null || fileName.isEmpty()){
             throw new HallImageNotFound(NO_USER_IMAGE_HAS_BEEN_UPLOADED);
         }
 
-        File file = new File(getUploadImageFilesPath() + fileName);
+        File file = new File(getUploadImageFilesPath() + "/" + fileName);
```
Comment (MichalNowakowski) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/cdi/FileUploadProcessor.java]: zamiast "/" uzyj File.pathSeparator albo File.separator

**Pull request merged**